index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav id="navbar" class="navbar bg-dark nav justify-content-center">
+    <li class="nav-item">
+      <a class="nav-link navbar-brand" href="#">Portfolio</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Next</a>
+    </li>
+    <li class="nav-item active">
+      <a class="nav-link" href="#">Home</span></a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#projects">Projects</a>
+    </li>
+  </nav>
+  <section id="welcome-section">
+    <div class="container">
+      <div class="card" style="width: 100%;">
+        <img src="https://res.cloudinary.com/anikriz/image/upload/v1658486635/microverse/portfolio-image_uupdpi.jpg" class="card-img-top" alt="image" style="height: 400px; object-fit: cover;">
+        <div class="card-body">
+          <h1 class="card-title">Portfolio file for Leehaney and Christian</h1>
+          <p class="card-text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam accumsan placerat nisi, ac tempus enim scelerisque vel. Fusce id libero ut nibh finibus facilisis. Integer tempor risus at ipsum tristique commodo. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Donec convallis tortor ut lectus auctor, varius egestas diam ornare. Etiam dignissim consequat dolor nec imperdiet. Nulla finibus sollicitudin volutpat. Suspendisse eget quam mollis, efficitur lorem sed, porta eros. Aliquam dictum pulvinar pretium. Pellentesque auctor suscipit euismod.            
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="projects">
+    <div class="container overflow-hidden text-center mt-5 mb-5">
+      <div class="row gy-5">
+        <div class="col project-tile">Check out the Git repo here:</div>
+        <div class="col"><a href="#">GitHub</a></div>
+      </div>
+    </div>
+  </section>
+  <div class="container overflow-hidden text-center mt-5 mb-5">
+    <div class="row gy-5">
+      <div class="col">
+        <a id="profile-link" href="https://www.freecodecamp.org" target="_blank">Profile Link</a>
+      </div>
+    </div>
```

Comment:
- [ ] [optional] Please note that this section should be enclosed within a footer tag:smiley: . More information can be found [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer).

Reply:
- [ Optional] Your comment is professional @leehaney254, Yet you haven't explained why in detail. Why should the `section` be enclosed with a `footer` tag? what is missing now? and what could be the result of implementing your suggestion?